src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery1 = new Lottery();
+        Lottery lottery2 = new Lottery();
+        Lottery lottery3 = new Lottery();
```

Comment:
We do not need 3 Lotteries like we do not need to visit 3 markets to buy 3 milk bottles.

Reply:
updated